cmd/mv.go

```diff
@@ -14,19 +14,53 @@
 
 package cmd
 
-import "github.com/spf13/cobra"
+import (
+	"fmt"
+	"os"
 
-func mv(cmd *cobra.Command, args []string) (err error) {
-	arg, err := makeRelocationArg(args[0], args[1])
-	if err != nil {
-		return
+	"github.com/dropbox/dropbox-sdk-go-unofficial/files"
+	"github.com/spf13/cobra"
+)
+
+func mv(cmd *cobra.Command, args []string) error {
+	var destination string
+	var argsToMove []string
+
+	if len(args) > 2 {
+		destination = args[len(args)-1]
+		argsToMove = args[0 : len(args)-1]
+	} else if len(args) == 2 {
+		destination = args[1]
+		argsToMove = append(argsToMove, args[0])
+	} else {
+		return fmt.Errorf("mv command requires a source and a destination")
+	}
+
+	mvErrors := []error{}
+	relocationArgs := []*files.RelocationArg{}
+
+	for _, argument := range argsToMove {
+		arg, err := makeRelocationArg(argument, destination+"/"+argument)
+		if err != nil {
+			relocationError := fmt.Errorf("Error validating move for %s to %s: %v", argument, destination, err)
+			mvErrors = append(mvErrors, relocationError)
+		} else {
+			relocationArgs = append(relocationArgs, arg)
+		}
+	}
+
+	for _, arg := range relocationArgs {
+		if _, err := dbx.Move(arg); err != nil {
+			moveError := fmt.Errorf("Move error: %v", arg)
+			mvErrors = append(mvErrors, moveError)
+		}
 	}
 
-	if _, err = dbx.Move(arg); err != nil {
-		return
+	for _, mvError := range mvErrors {
+		fmt.Fprintf(os.Stderr, "%v\n", mvError)
 	}
 
-	return
+	return nil
```

Comment:
My concern with that was that if some of the `mv`'s were successful then the program probably shouldn't have a `-1` exit status, which is how all cmd's with errors are being handled now (root.go:212). By the same logic I can understand returning an error if none were moved successfully. Of course that's strictly a design choice/opinion. I'm fine with however you want it.